### PR TITLE
Improve formatting for providers & accounts

### PR DESCRIPTION
**Pull request by lwander**
google-account

### CODE REVIEW

**Comment by lwander** [comment]
@duftler PTAL

**Comment by lwander** [comment]
@jtk54 PTAL

File: halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/providers/AbstractNamedProviderCommand.java [hunk 1]
```diff
@@ -18,7 +18,10 @@
 
 import com.netflix.spinnaker.halyard.cli.services.v1.Daemon;
 import com.netflix.spinnaker.halyard.cli.ui.v1.AnsiUi;
+import com.netflix.spinnaker.halyard.cli.ui.v1.AnsiFormatUtils;
+import com.netflix.spinnaker.halyard.config.model.v1.node.Account;
```
Comment (jtk54) [inline halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/providers/AbstractNamedProviderCommand.java]: Did you accidentally add imports for `Account` and `List`? I don't see them used in this diff...

Comment (lwander) [inline halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/providers/AbstractNamedProviderCommand.java]: Whoops, thanks

**Pull request merged**